satnogsclient/__init__.py

```diff
@@ -21,6 +21,3 @@
 
     if not GROUND_STATION_LON:
         raise Exception('GROUND_STATION_LON not configured')
-
-    if not GROUND_STATION_ELEV:
```

Comment:
Is there a reason why you are removing this check? ``GROUND_STATION_ELEV`` is required in the orbital calculations.